src/loac/loac_compute.h

```diff
@@ -0,0 +1,517 @@
+#pragma once
+
+#include <vector>
+#include <algorithm>
+#include <limits>
+#include <iostream>
+#include <iomanip>
+#include <unordered_map>
+#include <random>
+#include <queue>
+
+#include "../kmers.h"
+#include "../unionfind.h"
+
+#define size_k_max uint8_t
+
+#define RANDOM_SEED 0
+#define MAX_COUNT_WIDTH 12
+#define MAX_ITERS_WIDTH 3
```

Comment:
Nit: I would use in c++ `constexpr` instead of defines. E.g.
```suggestion
constexpr int RANDOM_SEED  = 0;
constexpr size_k_max MAX_COUNT_WIDTH = 12;
constexpr size_k_max MAX_ITERS_WIDTH = 3;
```
The only exception, in my opinion, would be if you set these via flags during compilation.